docs/global/modules/core/pages/reference/component-model-server.adoc

```diff
@@ -53,3 +53,40 @@ To reformat and add license header to all files run:
 Valid keys are keys starting with the PROTECTED_PREFIX ($$$).
 
 For details please refer to the OpenAPI documentation under `doc/model-server.json`.
+
+== JSON web tokens (JWT) based authorization
```

Comment:
The following settings touched by this PR lack documentation:

* MODELIX_GENERATE_FAKE_JWT
* KEYCLOAK_BASE_URL
* KEYCLOAK_REALM

Reply:
Added documentation for all three.